README.md

```diff
@@ -7,21 +7,32 @@
 * [Go - 1.15](https://golang.org/doc/go1.15)
 * [Docker](https://docs.docker.com/get-docker/)
 * [Docker Compose](https://docs.docker.com/compose/install/)
+* [NodeJS](https://nodejs.org/en/download/package-manager/)
 
+## Create necessary dot env files
+- Create `.env` file
+- Create env dependent files (depend on your env)
+  - `.env.release`
+  - `.env.debug` (for development)
+  - `.env.test`
+- Add values from `.env.example` (for env dependent variables eg. `DB_NAME` must add to dependent env  files)
```

Comment:
English:

```suggestion
- Add values from `.env.example` (env dependent variables eg. `DB_NAME` must be added to each dependent env file)
```

Reply:
Fixed in 30d1be4827d47295b8549c54aeef5c15ce040b76